### PR TITLE
Remove graphql and apollo.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -64,10 +64,5 @@ target 'OneSignalNotificationServiceExtension' do
 end
 
 target 'HeliumWalletWidgetExtension' do
-  pod "Apollo", '0.53.0'
```
Comment (tyler-whitman) [inline ios/Podfile:67]: Is there still code in the widget using apollo?

File: src/components/LedgerBurnModal.tsx [hunk 2]
```diff
@@ -77,12 +78,13 @@ const LedgerBurnModal = forwardRef(
...
             return
           }
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
           const payment = await signLedgerBurn(
             nextTransport,
             opts.unsignedTxn,
             opts.accountIndex,
           )
-          onConfirm({ txn: payment.txn, txnJson: opts.txnJson })
+          // onConfirm({ txn: payment.txn, txnJson: opts.txnJson })
```
Comment (tyler-whitman) [inline src/components/LedgerBurnModal.tsx:87]: What is this change for?

File: src/features/account/AccountActionBar.tsx [hunk 3]
```diff
@@ -52,18 +51,14 @@ const AccountActionBar = ({
   const anim = useRef(new Animated.Value(1))
   const { currentAccount } = useAccountStorage()
 
-  const { data: voteData } = useVotesQuery({
-    variables: { address: currentAccount?.address || '' },
-    skip: !currentAccount?.address,
-    fetchPolicy: 'cache-and-network',
-  })
+  const voteData = useMemo(() => {
+    // TODO: Are votes still a thing?
```
Comment (tyler-whitman) [inline src/features/account/AccountActionBar.tsx:55]: I dont think so haha? For another PR but we can probably take all the old vote stuff out..

File: src/hooks/useLedger.ts [hunk 4]
```diff
@@ -78,25 +75,16 @@ const useLedger = () => {
...
-          variables: { address },
-        })
-        balance = accountData?.account?.balance
-      } catch (e) {
-        const error = e as ApolloError
-        throw new Error(`Api failure - ${error.message}`)
-      }
-
+      const balance = 0
+      console.error('LEDGER NOT CURRENTLY SUPPORTED')
```
Comment (tyler-whitman) [inline src/hooks/useLedger.ts:79]: I wonder if we should take the ledger option out on import Account?

**Review by tyler-whitman — APPROVED** [review]

File: ios/Podfile (hunk 1 above)
Comment (Perronef5) [reply ios/Podfile:67]: Not anymore

File: src/features/account/AccountActionBar.tsx (hunk 3 above)
Comment (Perronef5) [reply src/features/account/AccountActionBar.tsx:55]: Yeah we can remove that

File: src/hooks/useLedger.ts (hunk 4 above)
Comment (Perronef5) [reply src/hooks/useLedger.ts:79]: We have a separate ticket for that

File: src/components/LedgerBurnModal.tsx (hunk 2 above)
Comment (Perronef5) [inline src/components/LedgerBurnModal.tsx:87]: Add TODO?

File: src/features/dappLogin/DappLoginScreen.tsx [hunk 5]
```diff
@@ -26,6 +23,39 @@ import {
   RootNavigationProp,
   RootStackParamList,
 } from '../../navigation/rootTypes'
+import { getKeypair } from '../../storage/secureStorage'
+
+// TODO: Verify crowdspot still works
```
Comment (Perronef5) [inline src/features/dappLogin/DappLoginScreen.tsx:28]: We should test this with crowdspot before merging

**Review by Perronef5 — APPROVED** [review]

File: src/features/dappLogin/DappLoginScreen.tsx (hunk 5 above)
Comment (matthewcarlreetz) [reply src/features/dappLogin/DappLoginScreen.tsx:28]: Tested. It works.

**Pull request merged**